packages/nextjs/components/scaffold-stark/Address.tsx

```diff
@@ -60,6 +60,11 @@ export const Address = ({
     checkSumAddress || address || "",
   );
 
+  const isValidHexAddress = (value: string): boolean => {
+    const hexAddressRegex = /^0x[0-9a-fA-F]+$/;
+    return hexAddressRegex.test(value);
+  };
+
```

Comment:
lets put it in the utils file (i think we have this util already). we don't want duplicates of the same function since they would be a pain to maintain

Reply:
I was given another recommendation that 0x be accepted as felt252 and that I should add a 0 to 0x to be accepted

Reply:
yep, aligned on that end

Reply:
please make sure `0x0x0` does not work

Reply:
Ok I'll check that it aligns with that